lib/src/theme/wolt_modal_sheet_default_theme_data.dart

```diff
@@ -148,7 +148,7 @@ class WoltModalSheetDefaultThemeData extends WoltModalSheetThemeData {
 
   /// Overrides the default value for [WoltModalSheet] clipBehavior.
   ///
-  /// If null, [WoltModalSheet] uses [Clip.none].
+  /// If null, [WoltModalSheet] uses [Clip.antiAliasWithSaveLayer].
```

Comment:
```suggestion
  /// Defaults to [Clip.antiAliasWithSaveLayer].
```


`Clip get clipBehavior => Clip.antiAliasWithSaveLayer;` is not nullable